codegen-test/model/misc.smithy

```diff
@@ -12,6 +14,8 @@ use aws.protocols#restJson1
 service MiscService {
     operations: [
         OperationWithInnerRequiredShape,
+        ResponseCodeRequired,
+        ResponseCodeDefault,
```

Comment:
We should also test that when neither `http.code` nor `httpResponseCode` are provided, we default to 200.